### ISSUE TITLE
zero dimension np.array is not supported in Phylanx

### ISSUE THREAD

**Issue by weilewei** [STEllAR-GROUP/phylanx#502]
for example:
the code
```
from phylanx.ast import *
import numpy as np


@Phylanx(debug=True)
def zero_dim_array(input_para):
    print("phylanx printing:", input_para)


a = np.array(123456)
print("python printing:", a)
zero_dim_array(a)
```
yields
```
define(
    zero_dim_array,
    input_para,
    (
        cout("phylanx printing:", input_para)
    )
)
Traceback (most recent call last):
  File "/root/PycharmProjects/nn/add_test.py", line 12, in <module>
    zero_dim_array(a)
  File "/root/.local/lib/python3.6/site-packages/phylanx-0.0.1-py3.6-linux-x86_64.egg/phylanx/ast/transformation.py", line 685, in __call__
    return et.eval(self.f.__name__, self.cs, *nargs)
ValueError: vector::reserve
python printing: 123456
```
Thanks for taking a look at!

**Comment by hkaiser** [comment]
nice catch!

**Comment by hkaiser** [comment]
This has been fixed by merging #503
